apps/docs/pages/guides/platform/custom-domains.mdx

```diff
@@ -89,6 +89,64 @@ As with the final activation stage of the process for setting up a custom domain
 
 Edge functions do not honor the custom domain setting and they still have to be invoked via the `foobarbaz.supabase.co` domain.
 
+# Vanity Subdomains
+
+<Admonition type="note">
+
+Vanity Subdomains are currently in beta, and are being slowly made available to projects. Contact Support through the Dashboard if you'd like to request early access.
+
+</Admonition>
+
+Vanity subdomains allow you to present a weaker branded experience, compared to custom domains. They allow you to host your services at a custom subdomain on Supabase, e.g. `my-example-brand.supabase.co`, instead of the default, randomly-assigned `foobarbaz.supabase.co`.
```

Comment:
```suggestion
Vanity Subdomains allow you to present a basic branded experience, compared to custom domains. They allow you to host your services at a custom subdomain on Supabase (e.g., `my-example-brand.supabase.co`) instead of the default, randomly-assigned `foobarbaz.supabase.co`.
```